### PR TITLE
Style updates

### PR DESCRIPTION
**Pull request by AlecKap**
____ Wrote Tests __x__ Implemented __x__ Reviewed

## Necessary checkmarks:

- [x] All Tests are Passing in all environments

- [x] The code will run locally

## Type of change

- [ ] New feature
- [ ] Bug Fix
- [x] Refactor

## Description of Change:

    description:

## Requested feedback:

    I'd like feedback on...

## Check the correct boxes

- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes

- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:

- [x] My code has no unused/commented out code
- [x] My code has no binding.pry's
- [x] I have reviewed my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

### CODE REVIEW

**Comment by AlecKap** [comment]
Update styling and refactor to utilize partials to follow SRP standards.

File: app/views/shared/_attendance_details.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<h3>Title: <%= attendance.meeting.title %></h3>
-<h3>Date: <%= pretty_date(attendance.attendance_time) %></h3>
-<h3>Attendance Time: <%= pretty_time(attendance.attendance_time) %> <%= link_to "Update Attendance Time", edit_attendance_path(attendance) if updatable %></h3>
+<h3>Title: <span style="font-weight: lighter; font-size: smaller;"><%= attendance.meeting.title %></span></h3>
```
Comment (margdaly) [inline app/views/shared/_attendance_details.html.erb:1]: Thank you for changing this - it is considerably more readable

File: app/views/shared/_zoom_attendance_table.html.erb [hunk 2]
```diff
@@ -1,34 +1,19 @@
-<h3>Title: <%= facade.meeting_title %></h3>
-<h3>Type: Zoom Meeting</h3>
-<h3>Meeting ID: <%= facade.meeting_id %></h3>
-<h3>Date: <%= facade.attendance_date %></h3>
-<h3>Attendance Time: <%= facade.attendance_time %> <%= link_to "Update Attendance Time", edit_attendance_path(facade.attendance) if facade.updatable %></h3>
-<p><strong>Present: <%= facade.status_count(:present) %>, Tardy: <%= facade.status_count(:tardy) %>, Absent: <%= facade.status_count(:absent) %></strong></p>
-<%= link_to "Transfer Student Attendances to Populi", new_attendance_populi_transfer_path(facade.attendance) if facade.updatable %>
+<div class="meeting-info">
```
Comment (margdaly) [inline app/views/shared/_zoom_attendance_table.html.erb:1]: nice abstraction here

**Review by margdaly — APPROVED** [review]
Great work with css. This makes it so much more user friendly

**Pull request merged**